modules/ROOT/pages/changelogs.adoc

```diff
@@ -1,6 +1,16 @@
 :description: This page lists all changes to status codes per Neo4j version.
 = Changes to status codes per Neo4j version
 
+== Neo4j 5.21
```

Comment:
remember to update this to correct version that this gets added in

Reply:
went into 5.21 behind feature flag, so it's correct unless we want to wait for the feature flag to be removed to merge/introduce the notifications here

Reply:
Updated to 5.24 so it's the same as the main docs PR